.github/workflows/unit_tests.yml

```diff
@@ -2,20 +2,16 @@ name: Unit Tests
 on:
   pull_request:
     branches:
-      - '*'
+      - "*"
   push:
     branches:
-      - '*'
+      - "master"
 jobs:
   build:
     runs-on: ubuntu-latest
     strategy:
       matrix:
         include:
-          - gemfile: gemfiles/rails_3.1.x.gemfile
-            ruby-version: 2.2
-          - gemfile: gemfiles/rails_3.2.x.gemfile
-            ruby-version: 2.2
```

Comment:
The CI setup just seems to generally fail with these two versions now (ie ruby setup can't handle it)

Reply:
Ok fine to remove then 👍